integration-test/src/it/depmgmt-strict-mode-exact-classifier/pom.xml

```diff
@@ -81,6 +81,14 @@
         <classifier>tck</classifier>
         <version>3.4.2</version>
       </dependency>
+      <dependency>
```

Comment:
This is added twice, once with default scope (compile) and once with scope test, but scope compile implies scope test.

Reply:
Correct - using it to test scope and ensure that it manages to track both otherwise identical dependencies.